hpx/parallel/executors/persistent_auto_chunk_size.hpp

```diff
@@ -0,0 +1,119 @@
+//  Copyright (c) 2016 Zahra Khatami
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+/// \file parallel/executors/persistent_auto_chunk_size.hpp
+
+#if !defined(HPX_PARALLEL_PERSISTENT_AUTO_CHUNK_SIZE_HPP)
+#define HPX_PARALLEL_PERSISTENT_AUTO_CHUNK_SIZE_HPP
+
+#include <hpx/config.hpp>
+#include <hpx/traits/is_executor_parameters.hpp>
+#include <hpx/runtime/serialization/serialize.hpp>
+#include <hpx/parallel/config/inline_namespace.hpp>
+#include <hpx/parallel/executors/executor_parameter_traits.hpp>
+#include <hpx/parallel/executors/executor_information_traits.hpp>
+#include <hpx/util/high_resolution_clock.hpp>
+#include <hpx/util/date_time_chrono.hpp>
+
+#include <cstddef>
+#include <algorithm>
+
+#include <boost/cstdint.hpp>
+
+namespace hpx { namespace parallel { HPX_INLINE_NAMESPACE(v3)
+{
+    ///////////////////////////////////////////////////////////////////////////
+    /// Loop iterations are divided into pieces and then assigned to threads.
+    /// The number of loop iterations combined is determined based on
+    /// measurements of how long the execution of 1% of the overall number of
+    /// iterations takes.
+    /// This executor parameters type makes sure that as many loop iterations
+    /// are combined as necessary to run for the amount of time specified.
+    ///
+    struct persistent_auto_chunk_size : executor_parameters_tag
+    {
+    public:
+        /// Construct an \a persistent_auto_chunk_size executor parameters object
+        ///
+        /// \note Default constructed \a persistent_auto_chunk_size executor parameter
+        ///       types will use 80 microseconds as the minimal time for which
+        ///       any of the scheduled chunks should run.
+        ///
+
+        persistent_auto_chunk_size()
+          : chunk_size_time_(0) , min_time_(80000)
+        {}
+
+        /// Construct an \a persistent_auto_chunk_size executor parameters object
+        ///
+        /// \param rel_time     [in] The time duration to use as the minimum
+        ///                     to decide how many loop iterations should be
+        ///                     combined.
+        ///
+
```

Comment:
Could you please update the document commentary above to reflect the function parameters?